pkg/e2e/verify/samesite_cookie.go

```diff
@@ -52,15 +52,22 @@ var _ = ginkgo.Describe(samesiteTestName, func() {
 func verifyVersion(h *helper.H) func() bool {
 	return func() bool {
 		unsupportedVersion := false
-		clusterVersionObj, currentVersion, err := h.GetClusterVersion()
+		clusterVersionObj, err := h.GetClusterVersion()
 		Expect(err).NotTo(HaveOccurred(), "failed getting cluster version")
 		Expect(clusterVersionObj).NotTo(BeNil())
 
 		// Get the cluster version and slice it, then convert the major/minor version to int Ex. majMinVersion := 46
-		splitVersion := strings.Split(currentVersion, ".")
+		splitVersion := strings.Split(clusterVersionObj.Status.Desired.Version, ".")
+
+		// Assume the len is <= 3 since semver major/minor could be something like 4.11.x
+		if len(splitVersion) > 3 && len(splitVersion) != 0 {
```

Comment:
I don't think this handles the (unlikely) case of len == {1,2,3}. 3 and 2 both don't cause panics, but 1 does.

Reply:
Good point, let me make it simpler.

Reply:
@whereswaldon I can just do this:
`		if len(splitVersion) != 2 || len(splitVersion) != 3 {
			return true
		}`
This way we explicitly expect a len or 2 or 3, anything else is unsupported.